Makefile.am

```diff
@@ -92,6 +90,7 @@ rpm: local
 
 release: tag
 	$(MAKE) archive
+	$(MAKE) -C po/ $(PACKAGE_NAME).pot-update
```

Comment:
The .pot file is no longer in the archive, since it isn't really doing anything useful in there, so it will no longer be generated as part of make archive. If you always run bumpver before release I suppose this isn't needed. I just put the .pot-update there because it's right before the .pot file gets pushed to Zanata.